.github/workflows/trivy.yml

```diff
@@ -0,0 +1,29 @@
+
+name: trivy
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  trivy:
+    name: trivy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.20.0
+        with:
+          scan-type: 'fs'
+          scan-ref: 'ops/dev/'
+          scanners: 'vuln,secret,config'
+          ignore-unfixed: true
```

Comment:
This line seems like it's causing a problem. We have critical vulnerabilities, based on the report, but the check is showing as "passing". Would bumping this to "false" cause it to fail as we would expect?

If not, we need to take a closer look at how this functions, because this doesn't block on critical issues at the moment.

Reply:
Changed ignore-unfixed and added exit-code, now moving forward this check will fail if a critical or high vulnerability is found. https://github.com/CDCgov/dibbs-cloud/actions/runs/9522759633/attempts/1